src/context/Global.tsx

```diff
@@ -146,6 +149,7 @@ const GlobalProvider = (props: { children: any }) => {
     });
 
     const [settingsMenu, setSettingsMenu] = createSignal<boolean>(false);
+    const [audio, setAudio] = createSignal<boolean>(false);
```

Comment:
```suggestion
    const [swapSuccessSound, setSwapSuccessSound = createSignal<boolean>(false);
```

This should be a persistent signal